aten/src/ATen/native/ForeachUtils.h

```diff
@@ -134,7 +134,18 @@ bool check_fast_path_restrictions(
         return false;
       }
       if (tensorLists[0][j].strides() != tensor_list[j].strides()) {
-        return false;
+        for (auto it1 = tensorLists[0][j].sizes().begin(),
```

Comment:
Can we rename it1, it2, and it3 to something that would help the reader more, maybe (feel free to adapt) ref_size, ref_stride, tensor_list_stride?

Reply:
addressed in b50604ae7ad86746350d16a8dc2d0f683d71f6cf